lisp/bazel-build.el

```diff
@@ -24,27 +24,42 @@
 
 ;;; Code:
 
+(defgroup bazel-build 'nil
+  "Package for editing, building, and running code using Bazel."
+  :link '(url-link "https://github.com/bazelbuild/emacs-bazel-mode")
+  :group 'languages)
+
+;;;###autoload (put 'bazel-build-bazel-command 'safe-local-variable 'file-executable-p)
+(defcustom bazel-build-bazel-command "bazel"
+  "Command that should be used to invoke Bazel."
+  :type 'string
```

Comment:
Should this rather be a list of command-line arguments, so that users can pass startup options if desired?

Reply:
Sounds reasonable, I’m not sure if the most idiomatic way is to put command and arguments in a variable or have something like `bazel-build-bazel-command` for the command itself and `bazel-build-bazel-command-args` for the arguments. I do believe I’ve seen the later in other packages, but I don’t have strong opinions myself.

Reply:
I also don't have strong opinions, but I think using a list here is a bit nicer because it means you can avoid introducing another variable.

Reply:
Done.